lib/auth-client.js

```diff
@@ -42,16 +42,33 @@ const config = require('./config.json');
  * must exactly match one of the registered URLs.
  * @param {Boolean} [options.testMode=false] - Launch Smartcar Connect in
  * [test mode](https://smartcar.com/docs/guides/testing/).
+ * testMode is now deprecated. Use mode instead.
+ * @param {String} [options.mode='live'] - Mode to Launch the Smartcar auth flow
+ * is one of [test|live|simulated]. Defaults to live.
```

Comment:
```suggestion
 * @param {String} [options.mode='live'] - Determine what mode Smartcar Connect should be
 * launched in. Should be one of test, live or simulated. 
```

- Prefer "Smartcar Connect" over "Smartcar auth flow"
- Defaults to live is redundant with `[options.mode='live']`